src/implementations/utils/cli-config.ts

```diff
@@ -0,0 +1,36 @@
+import { homedir } from "os";
+import fs from "fs";
+
+const fileDirectory = homedir + "/.config/open-attestation/";
+const fileName = "oacli-config.json";
```

Comment:
can we get some documentation written that it will create an oacli-config file?

does it need to be a json file though, can it just be a dotfile so that users wont really need to care about it

Reply:
ah yea, i missed out on the `.config`